promotion/claim.go

```diff
@@ -64,7 +64,7 @@ func blindCredsEq(a, b []string) bool {
 		return false
 	}
 	// a and b must have same values in same order
-	for i, _ := range a {
```

Comment:
Is this related?

Reply:
Got a linting comment on it when i ran the linter, so fixed.